sections/header.liquid

```diff
@@ -350,41 +350,43 @@
     {%- endif -%}
 
     <div class="header__icons">
-      <details-modal class="header__search">
-        <details>
-          <summary class="header__icon header__icon--search header__icon--summary link link--text focus-inset modal__toggle" aria-haspopup="dialog" aria-label="{{ 'general.search.search' | t }}">
```

Comment:
When I select `top center`, on desktop the search icon is moving to the left side. Is that expected? https://screenshot.click/27-06-kmcyk-9wh95.mp4

Reply:
Yep. This is the intended behaviour for the top-center setting.